### DISCUSSION TITLE
New Component: Tooltip

### DISCUSSION THREAD

**Discussion by Mil4n0r** [dxc-technology/halstack-react#1797] [Ideas] [answered]
Related to the issue #1794, it would also be interesting as an extra functionality for the new `Tag` component. It offers a lot of utility to add extra information and context in many of our existing components, while it would also improve the overall accesibility and space efficiency of our Design System.

**Comment by GomezIvann** [comment, marked as answer]
We are currently designing and developing this component so we can close this discussion!
